src/components/SVGImage.jsx

```diff
@@ -59,10 +77,21 @@ export default class SVGImage extends React.Component {
     }
   }
 
+  invertFilter() {
+    if (this.props.contrast) {
+      if (!document.getElementById('svg-invert-filter')) {
+        document.body.insertAdjacentHTML('afterbegin', INVERT);
```

Comment:
Adding the `#svg-invert-filter` SVG definition this way is functional, but it breaks the React pattern. Let me take a look and see if there's a way of doing this in a more React-ish way - maybe inserting the `<svg><defs>` in the SubjectViewer's SVG, or in the Classifier Page HTML or something.